README.md

```diff
@@ -172,11 +216,10 @@ const TVEventHandlerView: () => React.Node = () => {
 // Class based component
 
 class Game2048 extends React.Component {
-  _tvEventHandler: any;
+  _tvEventHandlerSubscription: any;
```

Comment:
Nit: Would be nice if we type it properly

Reply:
Good point, I'll have a look at doing that.

Reply:
Fixed